user-interface/7001-DATA-data_display.md

```diff
@@ -0,0 +1,50 @@
+# Data display
+
+This is a definition of some common data types and the rules about the display them. These are referenced in other acceptenace criteria to avoid repetition.
```

Comment:
```suggestion
This is a definition of some common data types and the rules about the displaying them. These are referenced in other acceptenace criteria to avoid repetition.
```